### DISCUSSION TITLE
rocksdb-doc-cn/doc/Persistent-Read-Cache

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/rocksdb-doc-cn#9] [General]
# rocksdb-doc-cn/doc/Persistent-Read-Cache

中文版rocksdb文档，感谢johnzeng https://wanghenshui.github.io/rocksdb-doc-cn/

https://wanghenshui.github.io/rocksdb-doc-cn/doc/Persistent-Read-Cache.html

**Comment by wanghenshui** [comment]
test
